app/templates/select-item-layout.hbs

```diff
@@ -1,5 +1,11 @@
 {{#if view.content.isGroupOption}}
   {{view.content.name}}
 {{else}}
-  {{yield}}
```

Comment:
This also doesn't make any sense logically because the `select-item` is used in the View helper.